3p/integration.js

```diff
@@ -199,6 +199,7 @@ import {openadstream} from '#ads/vendors/openadstream';
 import {openx} from '#ads/vendors/openx';
 import {opinary} from '#ads/vendors/opinary';
 import {outbrain} from '#ads/vendors/outbrain';
+import {piberica} from '#ads/vendors/piberica';
```

Comment:
Please remove changes to this file. This is not necessary for new networks.

Reply:
I just removed them. Thanks